include/functions.h

```diff
@@ -223,4 +224,6 @@ extern void sub_080806BC(u32, u32, u32, u32);
 
 extern void LoadObjPalette(Entity*, u32);
 
+extern void sub_080ADA04(OAMCommand*, void*);
```

Comment:
```suggestion

```
Could you put this in screen.h instead? Adding screen.h to functions.h will add that header as a unnecessary dependency to a ton of files.